eeos/src/main/java/com/blackcompany/eeos/target/application/support/AttendCountCalculate.java

```diff
@@ -0,0 +1,32 @@
+package com.blackcompany.eeos.target.application.support;
+
+import com.blackcompany.eeos.target.application.model.AttendModel;
+import com.blackcompany.eeos.target.application.model.AttendStatus;
+import java.util.List;
+import org.springframework.stereotype.Component;
+
+@Component
+public class AttendCountCalculate {
+
+	public Long attendCount(List<AttendModel> attendModels) {
+		return attendModels.stream()
+				.filter(attend -> AttendStatus.find(attend.getStatus()).equals(AttendStatus.ATTEND))
+				.count();
+	}
+
+	public Long absentCount(List<AttendModel> attendModels) {
```

Comment:
```suggestion
	public long countByStatus(AttendStatus status) {
        return records.stream()
                .filter(attend -> AttendStatus.find(attend.getStatus()).equals(status))
                .count();
    }
```
AttendStatus가 늘어났을 때도 추가적으로 변경하지 않아도 되도록 하는 건 어떠신가요~?

Reply:
오 좋은 방법이네요!